src/ScrollBox/ScrollBox.js

```diff
@@ -3,13 +3,13 @@ import cx from 'classnames'
 import Box from '../Box/Box'
 import './scrollbox.css'
 
-const ScrollBox = ({className, children, ...props}) => {
+const ScrollBox = ({className, scrollX = false, scrollY = false, children, ...props}) => {
   const classes = cx('u-scrollbox', className)
 
   return (
-    <Box className={classes} {...props}>
-      <Box className='u-scrollbox__wrapper'>
-        <Box className='u-scrollbox__container'>{children}</Box>
+    <Box height="full" width="full" overflow="hidden" className={classes} {...props}>
```

Comment:
oh i realized u-scrollbox never added full height before so I'll remove this prop